python_modules/dagster/dagster/_core/definitions/auto_materialize_policy.py

```diff
@@ -1,10 +1,17 @@
 import datetime
+from enum import Enum
 from typing import NamedTuple, Optional
 
 from dagster._annotations import experimental, public
 from dagster._serdes.serdes import whitelist_for_serdes
 
 
+class AutoMaterializePolicyType(Enum):
+    EAGER = "EAGER"
+    LAZY = "LAZY"
+    CUSTOM = "CUSTOM"
```

Comment:
see below, but I don't think we should inclue `CUSTOM` until we are sure we need it